src/pudl_rmi/connect_deprish_to_ferc1.py

```diff
@@ -658,100 +658,6 @@ def _allocate_col(
-    data's main data columns. Right now, this is hard coded to fail when there
-    are more inconsitent plants and utilities than are currently known.
-
-    TODO: I think :func:``data_col_test`` is a perfect candidate for the
-    decorators that let you test multiple inputs (different stages of the data
-    and different data columns) as well as output values (in this case the
-    number of known bad utilities and plants).
-
-    Args:
-        df1: One dataframe to sum and check consistency with ``df2``. Both
-            dfs should have columns ``data_col`` as well as identifying
-            columns: ``['report_date', 'data_source', 'utility_id_pudl',
-            'plant_id_eia']``
-        df2: Other dataframe to sum and check consistency against ``df1``.
-
-    """
-    util_bad, plant_bad = data_col_test(
-        df1=df1, df2=df2, data_col='plant_balance_w_common')
-
-    # known baddies. We need to track these down
-    assert(len(plant_bad) <= 53)
-    assert(len(util_bad) <= 88)
-
-    util_bad, plant_bad = data_col_test(
-        df1=df1, df2=df2, data_col='unaccrued_balance_w_common')
-
-    # known baddies. We need to track these down
-    assert(len(plant_bad) <= 35)
-    assert(len(util_bad) <= 70)
-
-
-def data_col_test(
-    df1: pd.DataFrame,
-    df2: pd.DataFrame,
-    data_col: str
-) -> (pd.DataFrame, pd.DataFrame):
-    """
-    Check consistency of a column at the plant and utility level of two dfs.
-
-    The sum of the ``data_col`` at both plant and utlity aggregations should be
-    the same in both dfs.
-
-    Args:
-        df1: One dataframe to sum and check consistency with ``df2``. Both
-            dfs should have columns ``data_col`` as well as identifying
-            columns: ``['report_date', 'data_source', 'utility_id_pudl',
-            'plant_id_eia']``
-        df2: Other dataframe to sum and check consistency against ``df1``.
-        data_col: data column to check. Column must be in both ``df1`` and
-            ``df2``.
-
-    """
-    util_test = gb_test(
-        df1,
-        df2,
-        data_col=data_col,
-        by=['report_year', 'data_source', 'utility_id_pudl']
-    )
-
-    logger.info(
-        f"Duke utility level data is consistent for {data_col}!! Go forth and "
-        "close them coal plants lil tables")
-
-    plant_test = gb_test(
-        df1,
-        df2,
-        data_col=data_col,
-        by=['report_year', 'data_source', 'utility_id_pudl', 'plant_id_eia']
-    )
-
-    util_bad = util_test[~util_test.match & util_test.match.notnull()]
-    plant_bad = plant_test[~plant_test.match & plant_test.match.notnull()]
-    logger.warning(
-        f"We have {len(util_bad)} utilities and {len(plant_bad)} "
-        f"plants who's {data_col} doesn't match the input."
-    )
-
-    # we know the duke utilities should be correct!
-    # assert(util_test.loc[2018, 'FERC', 90].match)
-    # assert(util_test.loc[2018, 'FERC', 97].match)
-    return util_bad, plant_bad
-
 
 def gb_test(
```

Comment:
I think anything that's only being used in the tests should be put in the test module. I think you may want to change the name not to start or end with `test` so that pytest doesn't think it's a test. Also a more descriptive name would be helpful.

Reply:
cool I moved everything over.